src/components/Modal/dialog.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { MODAL } from '../ThemeIdentifiers';
+import Helpers from './helpers';
+import Base from './base';
+import * as baseTheme from './Modal.scss';
+
+
+export interface Props {
+  children: any,
```

Comment:
1. No need for children.
2. Specify the types for onClose, closeOnBackground, and size.